src/api/Feed.js

```diff
@@ -526,7 +533,7 @@ class Feed extends Base {
      * @param {Function} errorCallback - the function which will be called on error
      * @return {void}
      */
-    createTaskNew = (
+    createTaskNew = async (
```

Comment:
let's keep this as classic callback style for now, since it doesn't return `Promise`

No use of `await` inside the function

Also see #LN546